sphinx/cmd/quickstart.py

```diff
@@ -357,6 +358,7 @@ def generate(d: Dict, overwrite: bool = True, silent: bool = False, templatedir:
     d.setdefault('extensions', [])
     d['copyright'] = time.strftime('%Y') + ', ' + d['author']
 
+    d["path"] = str(pathlib.Path(d['path']).resolve())
```

Comment:
Now sphinx does not use pathlib at all. So it would be better to use `path.abspath()` instead. What do you think?

Reply:
Oh yes, a much better solution!